src/sentry/static/sentry/app/styles/animations.jsx

```diff
@@ -53,3 +53,15 @@ export const slideInRight = keyframes`
     opacity: 1;
   }
 `;
+
+export const slideInUp = keyframes`
+  0% {
+    transform: translateY(10px);
```

Comment:
This seems pretty specific, does this belong in this `animations` module?

Reply:
I thought about that, but so did `slideInOver` I though, plus we were already importing some animations, so I just put it in there.